internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -71,17 +66,31 @@ func (provisioner *KopsProvisioner) CreateClusterInstallation(cluster *model.Clu
 			},
 		},
 		Spec: mmv1alpha1.ClusterInstallationSpec{
-			Size:                   installation.Size,
-			Version:                translateMattermostVersion(installation.Version),
-			Image:                  installation.Image,
```

Comment:
Doesn't look like we are setting the image anymore after the change.

Reply:
@gabrieljackson Sorry missed that somehow. Fixed.